### PR TITLE
Filter for containerslots-label

### PR DESCRIPTION
**Pull request by xFragger**
Was not able to add this to the existing PR #1809, don't know why, but here are my changes in one commit

### CODE REVIEW

**Comment by dongluochen** [comment]
@xFragger Please update document for your change https://github.com/docker/swarm/blob/master/docs/scheduler/filter.md .

**Comment by xFragger** [comment]
Hope my english is good enough, just have a look, i think it is okay this way. What do you think?

File: docs/scheduler/filter.md [hunk 1]
```diff
@@ -48,6 +49,9 @@ $ swarm manage --filter=health --filter=dependency
 
 When creating a container or building an image, you use a `constraint` or
 `health` filter to select a subset of nodes to consider for scheduling.
+If there are nodes in the swarm that have a label named `containerslots`
```
Comment (dongluochen) [inline docs/scheduler/filter.md]: In docs, we use `Swarm` instead of `swarm`.

File: docs/scheduler/filter.md [hunk 2]
```diff
@@ -48,6 +49,9 @@ $ swarm manage --filter=health --filter=dependency
...
+with a number-value, swarm will not launch more containers than given
```
Comment (dongluochen) [inline docs/scheduler/filter.md]: We refer label as key-value pair. 

> a label named `containerslots` with a number-value

=> a label with key as `containerslots` and a number-value

File: docs/scheduler/filter.md [hunk 3]
```diff
@@ -48,6 +49,9 @@ $ swarm manage --filter=health --filter=dependency
...
+with this number.
```
Comment (dongluochen) [inline docs/scheduler/filter.md]: => Swarm will not launch more containers than the given number.

File: docs/scheduler/filter.md [hunk 4]
```diff
@@ -175,6 +179,16 @@ The node `health` filter prevents the scheduler form running containers
 on unhealthy nodes. A node is considered unhealthy if the node is down or it
 can't communicate with the cluster store.
 
+### Use the containerslots filter
+
+You may give your docker nodes the containerslots label
```
Comment (dongluochen) [inline docs/scheduler/filter.md]: docker -> Docker

**Comment by xFragger** [comment]
Updates the docs, thanks for your feedback

**Comment by dongluochen** [comment]
@xFragger The docs are not updated. Did you forget to push your change?

**Comment by xFragger** [comment]
sorry, here it is

**Comment by dongluochen** [comment]
@SvenDowideit Do you know the reason of documentation test failure?

**Comment by SvenDowideit** [comment]
retest this please

**Comment by SvenDowideit** [comment]
@xFragger sorry, this PR needs a rebase to pick up some changes in the docs build.

**Comment by xFragger** [comment]
Okay, but i won't be able to Do this the next two Werks. So we have to wait or someone is willing to help out. On Vacation atm

**Comment by dongluochen** [comment]
@xFragger Thanks for your contribution. I'll take care of it.

**Pull request closed without merging**